server/eclipse-project/src/main/java/us/freeandfair/corla/model/DoSDashboard.java

```diff
@@ -59,6 +59,16 @@
   public static final int MIN_SEED_LENGTH = 20;
   
   /**
+   * The database stored precision for decimal types.
+   */
+  public static final int PRECISION = 6;
```

Comment:
Why the declaration here too?

Reply:
Is it something automatically consumed by Hibernate?

Reply:
It's a symbolic constant to allow us to change it more easily rather than hunting through the fields; also useful when we eventually add a risk limit for ballot polling audits, or different risk limits for state/county, etc.

Reply:
Of course, it wasn't actually used yet. But it is now.